app.js

```diff
@@ -3,12 +3,9 @@ const express = require('express');
 const path = require('path');
 const logger = require('morgan');
 const app = express();
+const indexRoute = require('./routes/index');
+const pizzaRoute = require('./routes/pizza')
```

Comment:
No need to make variables for something that's a one time use. You can just declare it inline like so:
```
app.use('/', require('./routes/index'));
```